src/Microsoft.ML.Core/Data/DataKind.cs

```diff
@@ -8,43 +8,39 @@
 namespace Microsoft.ML.Data
 {
     /// <summary>
-    /// Data type specifier used in text loader and type converters.
+    /// Data type specifiers used in text loader and type converters.
     /// </summary>
-    /// <remarks>
-    /// <list type="bullet">
-    /// <item><description><see cref="SByte"/>: 1-byte integer, type of <see cref="System.SByte"/>.</description></item>
-    /// <item><description><see cref="Byte"/>: 1-byte unsigned integer, type of <see cref="System.Byte"/>.</description></item>
-    /// <item><description><see cref="Int16"/>: 2-byte integer, type of <see cref="System.Int16"/>.</description></item>
-    /// <item><description><see cref="UInt16"/>: 2-byte usigned integer, type of <see cref="System.UInt16"/>.</description></item>
-    /// <item><description><see cref="Int32"/>: 4-byte integer, type of <see cref="System.Int32"/>.</description></item>
-    /// <item><description><see cref="UInt32"/>: 4-byte usigned integer, type of <see cref="System.UInt32"/>.</description></item>
-    /// <item><description><see cref="Int64"/>: 8-byte integer, type of <see cref="System.Int64"/>.</description></item>
-    /// <item><description><see cref="UInt64"/>: 8-byte usigned integer, type of <see cref="System.UInt64"/>.</description></item>
-    /// <item><description><see cref="Single"/>: 4-byte floating-point number, type of <see cref="System.Single"/>.</description></item>
-    /// <item><description><see cref="Double"/>: 8-byte floating-point number, type of <see cref="System.Double"/>.</description></item>
-    /// <item><description><see cref="String"/>: string, type of <see cref="System.String"/>.</description></item>
-    /// <item><description><see cref="Boolean"/>: boolean variable type, type of <see cref="System.Boolean"/>.</description></item>
-    /// <item><description><see cref="TimeSpan"/>: type of <see cref="System.TimeSpan"/>.</description></item>
-    /// <item><description><see cref="DateTime"/>: type of <see cref="System.DateTime"/>.</description></item>
-    /// <item><description><see cref="DateTimeOffset"/>: type of <see cref="System.DateTimeOffset"/>.</description></item>
-    /// </list>
-    /// </remarks>
     public enum DataKind : byte
     {
+        /// <summary><see cref="SByte"/>: 1-byte integer, type of <see cref="System.SByte"/>.</summary>
```

Comment:
You don't need `<see cref` in the summary for the thing you are providing a summary for:

```
/// <summary>1-byte integer, type of <see cref="System.SByte"/>.</summary>
```